src/common.jl

```diff
@@ -80,6 +74,33 @@ function solve{uType,tType,isinplace,algType<:ODEJLAlgorithm,F}(prob::AbstractOD
                           minstep=dtmin,
                           initstep=dt,
                           points=points)
+    elseif typeof(alg) <: ode4
+        ts,timeseries_tmp = ODE.ode4(f,u0,Ts,
+                          norm = norm,
+                          abstol=abstol,
+                          reltol=reltol,
+                          maxstep=dtmax,
+                          minstep=dtmin,
+                          initstep=dt,
+                          points=points)
+    elseif typeof(alg) <: ode4ms
+        ts,timeseries_tmp = ODE.ode4ms(f,u0,Ts,
+                          norm = norm,
+                          abstol=abstol,
+                          reltol=reltol,
+                          maxstep=dtmax,
+                          minstep=dtmin,
+                          initstep=dt,
+                          points=points)
+    elseif typeof(alg) <: ode4s
+        ts,timeseries_tmp = ODE.ode4s(f,u0,Ts,
+                          norm = norm,
+                          abstol=abstol,
+                          reltol=reltol,
+                          maxstep=dtmax,
+                          minstep=dtmin,
+                          initstep=dt,
+                          points=points)
     end
```

Comment:
Again, if you're willing to give up the type declaration, and since all of the algorithms accept a `kwargs...` parameter, you could simplify the code greatly:

```julia
function solve{uType,tType,isinplace,F}(prob::AbstractODEProblem{uType,tType,isinplace,F}, ode_alg,...* rest of args *)
    < snip >
    ts,timeseries_tmp = ode_alg(f,u0,Ts,
                           norm = norm,
                           abstol=abstol,
                           reltol=reltol,
                           maxstep=dtmax,
                           minstep=dtmin,
                           initstep=dt,
                           points=points)
```